packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -173,40 +274,69 @@ export const ContributionsEpicButtons = ({
         (secondaryCta?.type === SecondaryCtaType.Custom && isSupportUrl(secondaryCta.cta.baseUrl));
 
     return (
-        <div ref={setNode} css={buttonWrapperStyles} data-testid="epic=buttons">
+        <div ref={setNode} css={buttonWrapperStyles(showApplePayButton)} data-testid="epic=buttons">
             {!isReminderActive && (
                 <>
-                    <PrimaryCtaButton
-                        cta={getCta(cta)}
-                        tracking={tracking}
-                        numArticles={numArticles}
-                        amountsTestName={amountsTestName}
-                        amountsVariantName={amountsVariantName}
-                        countryCode={countryCode}
-                    />
-
-                    {secondaryCta?.type === SecondaryCtaType.Custom &&
-                    secondaryCta.cta.baseUrl &&
-                    secondaryCta.cta.text ? (
-                        <SecondaryCtaButton
-                            cta={secondaryCta.cta}
-                            tracking={tracking}
-                            countryCode={countryCode}
-                            numArticles={numArticles}
-                        />
+                    {showApplePayButton ? (
+                        <>
+                            <PrimaryCtaButtonApplePay
+                                cta={getCta(cta)}
+                                tracking={tracking}
+                                numArticles={numArticles}
+                                amountsTestName={amountsTestName}
+                                amountsVariantName={amountsVariantName}
+                                countryCode={countryCode}
+                            />
+                            <SecondaryCtaButtonApplePay
+                                cta={getCta(cta)}
+                                tracking={tracking}
+                                countryCode={countryCode}
+                                numArticles={numArticles}
+                            />
+                        </>
```

Comment:
I'm a bit confused here, `PrimaryCtaButtonApplePay` and `SecondaryCtaButtonApplePay` both return `ButtonApplePay`, will this render two Apple Pay buttons?

Reply:
OK, I see you're rendering `ButtonApplePay` in `SecondaryCtaButtonApplePay`, but it's actually rendering the button with other payment icons? I'd expect the `ButtonApplePay` component to only be used to render Apple Pay. If `showApplePay` is true why not use the standard `SecondaryCtaButton` to for the button with other payment icons?

Reply:
One issue with using `ButtonApplePay` to render both the Apple Pay button and the button with other payment icons is we'll be trigger the Apple Pay component click event whether it's the Apple Pay or the button with other payment icons being clicked.

Reply:
Yes agree the ApplePay component would be incorrectly triggered when the `SecondaryCtaButtonApplePay ` was pressed. `SecondaryCtaButtonApplePay ` now calls `SecondaryCtaButton` instead, this contains an icon for the `PaymentCardsSvg` component.  The `ButtonApplePay` has been simplified accordingly too.